contracts/src/v0.8/dev/ocr2dr/OCR2DROracle.sol

```diff
@@ -7,27 +7,40 @@ import "../AuthorizedReceiver.sol";
 import "../../ConfirmedOwner.sol";
 
 /**
- * @title OCR2DR oracle contract (stub for now)
+ * @title OCR2DR oracle contract
  */
 contract OCR2DROracle is OCR2DROracleInterface, AuthorizedReceiver, ConfirmedOwner {
```

Comment:
This still needs `getOperatorPublicKey` implemented for secrets, right?

Reply:
Done